views/uploadMenu.ejs

```diff
@@ -0,0 +1,20 @@
+<head>
+	<title>Upload Menu</title>
```

Comment:
Can we change to flash messages here for the errors?

Reply:
I tried to upload a non pdf file and ended up on a page just saying the error message and had to make my way back through url